### PR TITLE
use single tms api token and expose projects ids

### PR DESCRIPTION
**Pull request by camilo-stripe**
## Summary

- new token access is managed through LDAP https://ldapmanager.corp.stripe.com/groups/password-lokalise-api-token-manual
- no need to have encrypted values for the project ids, in fact is better to have it explicitly defined in case there's an issue (e.g better code searchibility)

## Motivation

https://jira.corp.stripe.com/browse/I18NENG-2817

## Testing

ran from laptop

### CODE REVIEW

**Comment by camilo-stripe** [comment]
r?

**Review by mludowise-stripe — COMMENTED** [review]
I'm no longer a reviewer for this repo so I'll leave approval up to @vardges-stripe, but script changes LGTM.

We should make sure that all iOS sdk deployers are in the appropriate ldap managed groups before merging so as not to disrupt deployments.

**Comment by mludowise-stripe** [comment]
Might want to also update our internal instructions to include getting ldap access to this group:
https://confluence.corp.stripe.com/display/MOBILE/Localization

**Review by vardges-stripe — APPROVED** [review]

**Pull request merged**
